localstack/services/apigateway/apigateway_listener.py

```diff
@@ -742,22 +801,40 @@ def invoke_rest_api_integration_backend(
                 target = kinesis_listener.ACTION_LIST_STREAMS
             else:
                 LOG.info(
-                    "Unexpected API Gateway integration URI '%s' for integration type %s",
-                    uri,
-                    integration_type,
+                    f"Unexpected API Gateway integration URI '{uri}' for integration type {integration_type}",
                 )
                 target = ""
 
-            # apply request templates
-            new_data = apply_request_response_templates(
-                data, integration.get("requestTemplates"), content_type=APPLICATION_JSON
-            )
+            try:
+                json_str = json.dumps(data) if isinstance(data, (dict, list)) else to_str(data)
+                json_str = apply_template(
+                    integration,
+                    "request",
+                    json_str,
+                    path_params=path_params,
+                    query_params=query_string_params,
+                    headers=headers,
+                )
+                payload = json.loads(json_str)
+            except Exception as e:
+                LOG.error("Unable to convert API Gateway payload to str", e)
```

Comment:
Same as above - would change the level to `info` or `warning` here.

Reply:
I'm using error here because I'm raising an error, but looking at the similar path above, we pretty much ignore it, which is wrong, causing an issue down the line.